go.mod

```diff
@@ -6,9 +6,10 @@ require (
 	github.com/cucumber/godog v0.11.0
 	github.com/hashicorp/go-hclog v0.15.0 // indirect
 	github.com/markbates/pkger v0.17.1
-	github.com/probr/probr-sdk v0.1.1
+	github.com/probr/probr-sdk v0.0.0-00010101000000-000000000000
+	// github.com/probr/probr-sdk v0.1.5
 	k8s.io/api v0.19.6
 )
 
 // For Development Only
-// replace github.com/probr/probr-sdk => ../probr-sdk
+replace github.com/probr/probr-sdk => ../probr-sdk
```

Comment:
Can't target sdk v0.1.6 until it's released (doing so earlier will cause the go library manager thingy to permenantly refuse that version... fun stuff)